src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -669,6 +669,11 @@ UCS_CLASS_INIT_FUNC(uct_rc_verbs_ep_t, const uct_ep_params_t *params)
         goto err_qp_cleanup;
     }
 
+    status = uct_rc_verbs_iface_common_prepost_recvs(iface);
```

Comment:
just noticed that even in this case we have to do some additional cleanup - `uct_ib_device_async_event_unregister`